lightstep.gemspec

```diff
@@ -17,6 +17,10 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
 
   spec.add_dependency 'concurrent-ruby', '~> 1.0'
+  spec.add_dependency 'google-protobuf', '~> 3.1'
+  spec.add_dependency 'google-protoc', '~> 3.0.0.alpha.6.0.0'
+  spec.add_dependency 'google-api-client', '~> 0.24.3'
+  spec.add_dependency 'googleapis-common-protos', '~> 1.3'
```

Comment:
Hard-tagging these at the patch-level (or minor, for that matter) here would prevent any downstream libraries/systems from using newer versions, right? Also, do you need the `protoc` gem as a runtime dep?

Reply:
Hi thanks for looking, and no probably not. I think I need to close this PR, it is a half finished work in progress. Apologies for leaving it up.